src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -67,6 +78,15 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
         <Text variant="sm" color="black60">
           {shippingOrigin}
         </Text>
+        {user && userHasLabFeature(user, "Avalara Phase 2") && (
+          <Text variant="sm">
+            {artworkPrice &&
+              `${priceLabel} ${appendCurrencySymbol(
+                artworkPrice.price,
+                currencyCode
```

Comment:
#non-blocking. Eventually we want to move the currency formatting logic to Metaphysics, instead of handling it on every client. @ashleyjelks has an [open PR](https://github.com/artsy/metaphysics/pull/3692) which can be a good pattern to follow.

Reply:
Great, I will take a look!